### DISCUSSION TITLE
sort by lastest and unread

### DISCUSSION THREAD

**Discussion by tala-coder** [dailydotdev/daily#935] [Feature Requests]
I often use daily.dev, does it have an option to sort unviewed content and sort by post date?

**Comment by idoshamun** [comment]
Hi @tala-coder,

We have a sort by date feature! Please read more about it here: https://docs.daily.dev/docs/key-features/feeds#sorting-by-date
If you sort by date you will see content you already viewed, if you use the algorithmic feed we hide this content from the feed.
I hope it makes sense.

**Reply by tala-coder** [reply]
thank you, I didn't notice that
